src/main/java/org/jitsi/videobridge/LipSyncHack.java

```diff
@@ -255,16 +255,15 @@ void onRTPTranslatorWillWriteAudio(
             return;
         }
 
-        List<MediaStreamTrackDesc> sourceTracks
-            = source.getEndpoint().getMediaStreamTracks();
-        if (sourceTracks.isEmpty())
+        MediaStreamTrackDesc[] sourceTracks
```

Comment:
ah bummer, we can't keep this as a `List`?

Reply:
What I did in the reverted commit was cache a copy of the array as a `List`. Now I avoided the extra copy. If we want `List` though (which I would prefer), I think we should make MediaStreamTrackReceiver hold and/or return a list.